lib/commands/check-versions.js

```diff
@@ -0,0 +1,61 @@
+const Input = require('../utility/input');
+const Output = require('../utility/output');
+const Repository = require('../utility/repository');
+const AdmZip = require('adm-zip');
```

Comment:
@thomasmars why do we need to use this instead of the already required `archiver`?